src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax;
+
+import core.basesyntax.converter.DataConverter;
+import core.basesyntax.converter.DataConverterImpl;
+import core.basesyntax.io.FileReader;
+import core.basesyntax.io.FileReaderImpl;
+import core.basesyntax.io.FileWriter;
+import core.basesyntax.io.FileWriterImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.operation.BalanceOperation;
+import core.basesyntax.operation.OperationHandler;
+import core.basesyntax.operation.OperationStrategy;
+import core.basesyntax.operation.OperationStrategyImpl;
+import core.basesyntax.operation.PurchaseOperation;
+import core.basesyntax.operation.ReturnOperation;
+import core.basesyntax.operation.SupplyOperation;
+import core.basesyntax.service.ShopService;
+import core.basesyntax.service.ShopServiceImpl;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import report.ReportGenerator;
+import report.ReportGeneratorImpl;
+
+public class Main {
+    private static final String READ_PATH = "recources/reportToRead.csv";
+    private static final String WRITE_PATH = "recources/finalReport.csv";
```

Comment:
use the system resources directory. the path should be src/main/resources/reportToRead.csv